newsfragments/3232.removed.md

```diff
@@ -0,0 +1 @@
+Remove all functionality deprecated in PyO3 0.18.
```

Comment:
Can we mention the `#[ærgs(..)]` syntax explicitly here as the headline item?

Reply:
Good idea, done.